frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -246,7 +244,7 @@ class DashboardHeader extends Component {
       databases,
     } = this.props;
 
-    const canEdit = dashboard.can_write && isEditable && !!dashboard;
+    const canEdit = dashboard.can_write;
```

Comment:
Since `isEditable` is outdated and I've removed it, we can also remove `!!dashboard` since it doesn't make much sense because we know `dashboard` exists from `dashboard.can_write` alone.